src/functions-and-arrays.js

```diff
@@ -1,19 +1,126 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(a, b) {
+  return Math.max(a, b)
+}
+
+maxOfTwoNumbers(2, 1);
+
+
 // Iteration #2: Find longest word
+
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(words) { 
+  var longestWord = "";
+  if (words.length === 0) {
+    return null;
+  }
+  if (words.length === 1) {
+    return words[0];
+  }
+ 
+  for(var i = 0; i < words.length; i++){
+    if(words[i].length > longestWord){
```

Comment:
here you need to compare the length of both strings/words  and then make the longestWord a new strring so something like this 
```
    if (words[i].length > longestWord.length) {
      longestWord = words[i];
    }
  
  ```